dash-info.yaml

```diff
@@ -1,10 +1,2612 @@
-pkg_help_description: >
+pkg_help_description: >-
     Dash is a web application framework that
     provides pure Python and R abstraction around HTML, CSS, and
     JavaScript. Instead of writing HTML or using an HTML
     templating engine, you compose your layout using R
     functions within the dashHtmlComponents package. The
     source for this package is on GitHub:
     plotly/dash-html-components.
-pkg_help_title: >
+pkg_help_title: >-
     Vanilla HTML Components for Dash
+r_examples:
+    - name: htmlA
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlA(children='Link to external site',
+                      href='https://plot.ly',
+                      target='_blank')
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlAbbr
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+               htmlDiv(list(
+                 htmlAbbr(children='\U{1F50D} Hover over this line for a few seconds and see the text box appear...',
+                          title='Hello! htmlAbbr at work!')
+                  )
+               )
+            )
+
+            app$run_server()
+    - name: htmlAcronym
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlAcronym(children='Mouse over these words to see the acronym for \'as soon as possible\'.',
+                title='ASAP')
```

Comment:
htmlAcronym (and htmlAbbr above) should have their props reversed - children is the short text that's always shown, title is the longer explanation of it. https://www.w3docs.com/learn-html/html-acronym-tag.html

Reply:
Good point, will fix this.

Reply:
fixed in b4e683f